source/docs/articles/going-live.md

```diff
@@ -5,204 +5,56 @@ category:
-
-Finally, you should load test your Live environment to make sure everything is optimally configured.
-
-<table class=table>
-<tbody>
-		<tr>
-			<th class="complete">Complete</th>
-			<th class="action">Action</th>
-			<th class="help">Help</th>
-		</tr>
-		<tr>
-			<td class="complete">[ ]</td>
-			<td class="action">Disable development modules &amp; plugins</td>
-			<td>Ex: devel, examples, generate_errors, views_ui</td>
-		</tr>
-		<tr>
-			<td class="complete">[ ]</td>
-			<td class="action">Enable New Relic</td>
-			<td class="help"><a href="/docs/articles/sites/newrelic/new-relic-performance-analysis/">Using New Relic on Pantheon</a></td>
-		</tr>
-		<tr>
-			<td class="complete">[ ]</td>
-			<td class="action">Optimize Performance Settings</td>
-			<td class="help"><a href="/docs/articles/drupal/drupal-7-performance-and-caching-settings/">Performance and caching settings</a></td>
-		</tr>
-		<tr>
-			<td class="complete">[ ]</td>
-			<td class="action">Ensure Varnish caching works</td>
-			<td class="help"><a href="/docs/articles/sites/varnish/">Varnish caching for high performance</a></td>
-		</tr>
-					<tr>
-			<td class="complete">[ ]</td>
-			<td class="action">Load test Live environment</td>
-			<td class="help"><a href="/docs/articles/load-and-performance-testing/">Load and performance testing</a></td>
-		</tr>
-	</tbody>
-	</table>
-
-## Select a Paid Plan
-
-Once you're satisfied with your site configuration, you'll need to select a paid plan. Among other reasons, free sandbox sites can't have custom domains. You will need to add a credit card to either the site or the account to set up billing.
-
-<table class=table>
-<tbody>
-		<tr>
-			<th class="complete">Complete</th>
-			<th class="action">Action</th>
-			<th class="help">Help</th>
-		</tr>
-		<tr>
-			<td class="complete">[ ]</td>
-			<td class="action">Select a paid plan</td>
-			<td class="help"><a href="/docs/articles/sites/settings/selecting-a-plan/">Selecting a plan</a></td>
-		</tr>
-	</tbody>
-	</table>
+## Best Practices for Launching Sites Checklist
+&nbsp;&nbsp;&nbsp;<span class="glyphicon  glyphicon-unchecked" aria-hidden="true"></span> [Deploy to the Live Environment](/docs/articles/sites/code/using-the-pantheon-workflow/#3.-deploy-code-to-live)<br>
+&nbsp;&nbsp;&nbsp;<span class="glyphicon  glyphicon-unchecked" aria-hidden="true"></span> [Select a Plan for Your Site](/docs/articles/sites/settings/selecting-a-plan/)<br>
+&nbsp;&nbsp;&nbsp;<span class="glyphicon  glyphicon-unchecked" aria-hidden="true"></span> [Schedule Daily and Weekly Backups](#schedule-backups)<br>
+**Configure Site Monitoring Services:**<br>
+&nbsp;&nbsp;&nbsp;<span class="glyphicon  glyphicon-unchecked" aria-hidden="true"></span> [Enable New Relic for Performance Analysis](#monitoring-services)<br>
+&nbsp;&nbsp;&nbsp;<span class="glyphicon  glyphicon-unchecked" aria-hidden="true"></span> [Create Site Alerts with Pingdom](#monitoring-services) (Optional)<br>

```

Comment:
@JessiFischer  @suzannealdrich  @erik-pantheon Wouldn't you want to do this after updating DNS?